dsl_parser/functions.py

```diff
@@ -1072,6 +1072,46 @@ def evaluate(self, handler):
         return handler.get_environment_capability(self.capability_path)
 
 
+@register(name='string_find', func_eval_type=HYBRID_FUNC)
+class StringFind(Function):
+    def __init__(self, args, **kwargs):
+        self.haystack = None
+        self.needle = None
+        super(StringFind, self).__init__(args, **kwargs)
+
+    def parse_args(self, args):
+        if len(args) < 2:
```

Comment:
```suggestion
        if len(args) != 2:
```
error message says "exactly two" so let's make it exactly two :P